Sasquatch/SasquatchObjC/UserSettings.h

```diff
@@ -0,0 +1,27 @@
+//
```

Comment:
You might want to run the formatter on the UserSettings files.

Reply:
removed the files